manifest.json

```diff
@@ -0,0 +1,72 @@
+{
+  "repos": [
+    {
+      "name": "microsoft/dotnet-framework-samples",
+      "readmePath": "README.DockerHub.md",
+      "images": [
+        {
+          "sharedTags": {
+          },
+          "platforms": [
```

Comment:
This manifest doesn't look correct?  Have you run the image builder tool on this manifest?  I think it will fail the validation phase.

Each .NET version should be listed as a separate Image.  3.5 and 4.7.1 should each have two platforms and the 3.5 and 4.7.1 tags should be shared tags and not platform tags.  Furthermore, I would consider the standalone versions to be shared tags for all versions even though we are not building them for both server versions.  This would better support adding both versions if needed in the future.